server/src/main/java/org/apache/seata/server/cluster/manager/ClusterConfigWatcherManager.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.seata.server.cluster.manager;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Queue;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+import javax.annotation.PostConstruct;
+import javax.servlet.AsyncContext;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.seata.common.thread.NamedThreadFactory;
+import org.apache.seata.common.util.CollectionUtils;
+import org.apache.seata.server.cluster.listener.ClusterConfigChangeEvent;
+import org.apache.seata.server.cluster.listener.ClusterConfigChangeListener;
+import org.apache.seata.server.cluster.watch.ConfigWatcher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+
+/**
+ *
+ * The type of cluster config watcher manager.
+ */
+@Component
+public class ClusterConfigWatcherManager implements ClusterConfigChangeListener {
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private static final Map<String/*namespace*/, Map<String/*dataId*/, Queue<ConfigWatcher<?>>>> WATCHERS = new ConcurrentHashMap<>();
+
+    private final ScheduledThreadPoolExecutor scheduledThreadPoolExecutor =
+            new ScheduledThreadPoolExecutor(1, new NamedThreadFactory("long-polling", 1));
+
+    @PostConstruct
+    public void init() {
+        // Responds to monitors that time out
+        scheduledThreadPoolExecutor.scheduleAtFixedRate(() -> {
```

Comment:
Why not use scheduleWithFixedDelay?

Reply:
Is one second enough?

Reply:
Change to scheduleWithFixedDelay to avoid timeout.
This depends on the number of clients which are subscribing to the configuration from this server, a longer period of time may result in clients not being able to perceive configuration changes in a timely manner.